docs/reference/builder.md

```diff
@@ -45,22 +45,25 @@ Dockerfile.
 >the build to transfer the entire contents of your hard drive to the Docker
 >daemon. 
 
-To use a file in the build context, the `Dockerfile` refers to the file with
-an instruction, for example,  a `COPY` instruction. To increase the build's
+To use a file in the build context, the `Dockerfile` refers to the file specified
+in an instruction, for example,  a `COPY` instruction. To increase the build's
 performance, exclude files and directories by adding a `.dockerignore` file to
 the context directory.  For information about how to [create a `.dockerignore`
 file](#dockerignore-file) see the documentation on this page.
 
 Traditionally, the `Dockerfile` is called `Dockerfile` and located in the root
 of the context. You use the `-f` flag with `docker build` to point to a Dockerfile
-anywhere in your file system.
+elsewhere in your file system.
```

Comment:
I think "anywhere" is good here, because it more clearly indicates that it doesn't have to be within the "build context"